deployment/updates/scripts/xml/notifications/2019_05_20_update_player_slide_module_notifications_include_select_poll_state_tag.template.xml

```diff
@@ -0,0 +1,91 @@
+		<sessionType>2</sessionType>
+	</session>
+
+	<multirequest>
+
+		<request service="eventNotificationTemplate" action="listAction" plugin="eventNotification" partnerId="5872">
+			<filter objectType="KalturaEventNotificationTemplateFilter">
+				<systemNameEqual>SLIDE_VIEW_CHANGE_CODE_CUE_POINT</systemNameEqual>
+				<typeEqual>pushNotification.Push</typeEqual>
+			</filter>
+			<pager objectType="KalturaFilterPager">
+				<pageSize>1</pageSize>
+			</pager>
+		</request>
+
+		<request service="eventNotificationTemplate" action="update" plugin="eventNotification" partnerId="5872">
+			<id>{1:result:objects:0:id}</id>
+			<template objectType="KalturaPushNotificationTemplate">
+				<name>SLIDE_VIEW_CHANGE_CODE_CUE_POINT</name>
+				<systemName>SLIDE_VIEW_CHANGE_CODE_CUE_POINT</systemName>
+				<description>Push notification template player slide view code cue point is added</description>
+				<automaticDispatchEnabled>1</automaticDispatchEnabled>
+				<eventType>5</eventType> <!-- EventNotificationEventType::OBJECT_CREATED -->
+				<eventObjectType>cuePointEventNotifications.CuePoint</eventObjectType>
+				<eventConditions objectType="array">
+					<item objectType="KalturaOrCondition">
+						<conditions objectType="array">
+							<item objectType="KalturaFieldMatchCondition">
+								<description>Cue-point type equals</description>
+								<field objectType="KalturaEvalStringField">
+									<code>$scope->getObject()->getType()</code>
+								</field>
+								<values objectType="array">
+									<item objectType="KalturaStringValue">
+										<value>{trigger_cuepoint}</value>
+									</item>
+								</values>
+							</item>
+						</conditions>
+					</item>
+					<item objectType="KalturaFieldMatchCondition">
+						<description>Tags_Like</description>
+						<field objectType="KalturaEvalStringField">
+							<code>$scope->getObject()->getTags() ? ( count(array_intersect(array("select-poll-state","remove-selected-thumb","change-view-mode"), explode(",", $scope->getObject()->getTags()))) ? 1 : 0 ) : 0</code>
+						</field>
+						<values objectType="array">
+							<item objectType="KalturaStringValue">
+								<value>1</value>
+							</item>
+						</values>
+					</item>
+				</eventConditions>
+				<queueNameParameters objectType="array">
+					<item objectType="KalturaPushEventNotificationParameter">
+						<key>entryId</key>
+						<description></description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getObject()->getEntryId()</code>
+						</value>
+					</item>
+				</queueNameParameters>
+				<userParameters objectType="array">
+					<item objectType="KalturaEventNotificationParameter">
+						<key>trigger_cuepoint</key>
+						<description>Cue-point type that triggers the event (Code type only), See:KalturaCuePointType:/api_v3/testmeDoc/enums/KalturaCuePointType.html</description>
+						<value objectType="KalturaStringValue">
+							<value>10042</value>
```

Comment:
This cue point type value is env specific (dynamic enum) you can't use it in the template.